tweetflask.py

```diff
@@ -79,11 +99,17 @@ def errors(status, detail):
 if __name__ == '__main__':
     with open('config.yaml', 'r') as yfile:
         yamldata = yaml.load(yfile)
-    context = ssl.SSLContext(ssl.PROTOCOL_TLSv1_2)
+    secureProtocol = yamldata['server']['secureProtocol']
+    try:
+        context = ssl.SSLContext(getattr(ssl, secureProtocol))
+    except AttributeError as e:
+        exit("Invalid secureProtocol")
```

Comment:
I followed your [config example](https://github.com/osu-mist/tweet-api/pull/6/commits/c5315fe4062d99faf4c6fd7b9e2cf05d6cc15889#diff-78873141e614d1f00afaed15fa9dc032R8) to set up the `secureProtocol` but I was not able to start the app and the only thing I got was `Invalid secureProtocol` error message. Could you take a look?